packages/react-pose/src/components/Transition/PoseGroup.tsx

```diff
@@ -3,14 +3,12 @@ import { Props } from './types';
 import Transition from './';
 
 // TODO: Deprecate, add notice about Transition and flipMove: true opt-in
-export default class PoseGroup extends React.PureComponent<Props> {
+export default class PoseGroup extends React.Component<Props> {
```

Comment:
Same change as you have done to `Transition`, as its component that receives children (that won't evey be the same from reference equality stand of view) it's unnecessary here